tests/System.Buffers.Experimental.Tests/RangeTests.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Buffers.Tests
+{
+    public class RangeTests
+    {
+        [Fact]
+        public void Basics() {
+
+            for (int first = 0; first < 10; first++)
```

Comment:
If first is allowed to be negative, can we start the for loop from -10 instead of 0?

Reply:
will fix.